pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -372,6 +370,25 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 	return false, nil
 }
 
+func compareStatusAndPrint(newCurrNodeState *operatorv1.NodeStatus, currNodeState *operatorv1.NodeStatus) {
+	compareLog := fmt.Sprintf("A node named %q is set to target revision %d and has current revision %d.", newCurrNodeState.NodeName, newCurrNodeState.TargetRevision, newCurrNodeState.CurrentRevision)
+	if !equality.Semantic.DeepEqual(currNodeState.LastFailedRevision, newCurrNodeState.LastFailedRevision) {
+		compareLog += fmt.Sprintf(" The last failed revision changed from %d to %d.", currNodeState.LastFailedRevision, newCurrNodeState.LastFailedRevision)
+	} else {
+		compareLog += fmt.Sprintf(" The last failed revision remains %d.", newCurrNodeState.LastFailedRevision)
+	}
+	if newCurrNodeState.LastFailedRevisionErrors == nil {
+		compareLog += " There are zero errors of failed revisions."
+	} else {
```

Comment:
you chain if-else in Go:
```
if ... {
} else if ... {
} else {
}
```

Reply:
Oh yea. I put it where it made sense to me